src/main/java/seedu/multitasky/logic/parser/ListCommandParser.java

```diff
@@ -25,28 +29,81 @@ public ListCommand parse(String args) throws ParseException {
         if (trimmedArgs.isEmpty()) {
             return new ListCommand();
         }
-
-        // Flags are delimited by whitespace
-        final String[] flags = trimmedArgs.split("\\s+");
-        if (!hasValidFlagCombination(flags)) {
+        // Preamble is necessary due to how ArgumentTokenizer works
```

Comment:
ah yes, its because a flag is found by searching for " " + Prefix.toString() + " " right now, so without the " " at the start of the trimmedArgs the tokenizer will think that the first word is part of the preamble and not a flag.